abb_irb4400_support/launch/robot_state_visualize_irb4400l30.launch

```diff
@@ -0,0 +1,27 @@
+<!--
+  Manipulator specific version of the state visualizer.
+
+  Defaults provided for IRB 4400/L30:
+   - J23_coupled = true
+
+  Usage:
+    robot_state_visualize_irb4400l30.launch robot_ip:=<value>
+-->
+<launch>
+  <arg name="robot_ip" />
+  <arg name="J23_coupled" default="true" />
```

Comment:
I'm sure you have, but just to be sure: you have verified this with robot studio or a real robot?